src/events/client/voiceChannelLeave.ts

```diff
@@ -9,12 +9,19 @@ export default async function voiceChannelLeaveHandler(member: Eris.Member, oldC
         return;
     }
 
+    if (oldChannel.id !== gameSession.voiceChannelID) {
+        return;
+    }
+
     if (checkBotIsAlone(guildID)) {
         gameSession.endSession();
         return;
     }
 
-    if (!gameSession.finished) {
-        gameSession.updateOwner();
+    if (gameSession.finished) {
```

Comment:
Let's combine this check with `if (!gameSession)` at the beginning for all of our voice channel handlers